### PR TITLE
[NOD-754] Fix staticcheck errors

### PR DESCRIPTION
**Pull request by someone235**

### CODE REVIEW

File: mempool/mempool.go [hunk 1]
```diff
@@ -1029,28 +1033,6 @@ func (mp *TxPool) maybeAcceptTransaction(tx *util.Tx, rejectDupOrphans bool) ([]
...
-// such as rejecting duplicate transactions, ensuring transactions follow all
-// rules, detecting orphan transactions, and insertion into the memory pool.
-//
-// If the transaction is an orphan (missing parent transactions), the
-// transaction is NOT added to the orphan pool, but each unknown referenced
-// parent is returned. Use ProcessTransaction instead if new orphans should
-// be added to the orphan pool.
-//
-// This function is safe for concurrent access.
-func (mp *TxPool) MaybeAcceptTransaction(tx *util.Tx, isNew bool) ([]*daghash.TxID, *TxDesc, error) {
```
Comment (dereeno) [inline mempool/mempool.go]: !

File: config/config.go [hunk 2]
```diff
@@ -591,7 +582,7 @@ func loadConfig() (*Config, []string, error) {
 	}
 
 	// Disallow 0 and negative min tx fees.
-	if activeConfig.MinRelayTxFee <= 0 {
+	if activeConfig.MinRelayTxFee == 0 {
```
Comment (dereeno) [inline config/config.go]: !

File: database/ffldb/dbcache.go [hunk 3]
```diff
@@ -24,19 +24,6 @@ const (
 	// threshold in between database cache flushes when the cache size has
 	// not been exceeded.
 	defaultFlushSecs = 300 // 5 minutes
-
-	// ldbBatchHeaderSize is the size of a leveldb batch header which
```
Comment (svarogg) [inline database/ffldb/dbcache.go]: If you haven't already - Please check where was that used, and maybe we want to use it ourselves?

Comment (someone235) [reply database/ffldb/dbcache.go]: It was never used

**Pull request merged**